tasks/components_templates/bundle_test.go.tmpl

```diff
@@ -10,13 +10,5 @@ import (
 )
 
 func TestBundleDependencies(t *testing.T) {
-	require.NoError(t, fx.ValidateApp(
-		// instantiate all of the ${BUNDLE_NAME} components, since this is not done
-		// automatically.
-		// fx.Invoke(func(<component1>) {}),
-		// fx.Invoke(func(<component2>) {}),
-
-		//fx.Supply(BundleParams{}),
-		//Bundle
-        ))
+	fxutil.TestBundle(t, Bundle, EXTERNAL_DEPENDENCIES)
```

Comment:
❓ As everything was commented in `fx.ValidateApp`, it seems you are here changing the behaviour of this test. Is it expected?

Reply:
Note this is a template to generate code. Do you think having the code below would be better?
```
fxutil.TestBundle(
         t, 
        Bundle, 
       // EXTERNAL_DEPENDENCIES
)
```

Reply:
Sorry I missed it was a template. As a consequence I think it's ok, thanks for your reply